src/change_stream.ts

```diff
@@ -726,6 +726,16 @@ export class ChangeStream<
     }, callback);
   }
 
+  async *[Symbol.asyncIterator](): AsyncIterator<TChange, void> {
```

Comment:
Can we add a test that demonstrates that when we attempt to iterate a closed change stream, nothing happens?